test/integration/sign-up-with-id-test.js

```diff
@@ -15,8 +16,10 @@ var options = {
 
 test('sign up with id', function (t) {
   store.clear()
-  t.plan(5)
+  t.plan(6)
 
+  // mocks
```

Comment:
good idea to add a comment here! Maybe let’s make it a bit more expressive, something like "mock the Date object to always return Jan 1, 2017 0:00:00"